healpix_alchemy/util.py

```diff
@@ -0,0 +1,46 @@
+import typing
+
+from sqlalchemy.ext.declarative import declared_attr
+
+
+class InheritTableArgs:
```

Comment:
What are the other target classes you envision extending this class?

Reply:
There are three other mixins that I am refactoring into this package from https://github.com/growth-astro/healpix-intersection-example: Tile and Region.